pkg/local_object_storage/metabase/db_test.go

```diff
@@ -29,7 +28,7 @@ func (s epochState) CurrentEpoch() uint64 {
 		return s.e
 	}
 
-	return math.MaxUint64
+	return 0
```

Comment:
I've thought that default must be a big enough epoch, why do we need this?

Reply:
Shouldn't all tests fail then?
Let it be clear: we use 0 here (epoch source), thus all objects are expired by default. Right?